### ISSUE TITLE
Can't connect to a url to get list

### ISSUE THREAD

**Issue by danielloader** [Tantalor93/dnspyre#110]
Reproduction:
```
docker run  tantalor93/dnspyre:v2.12.1 --server 8.8.8.8 https://raw.githubusercontent.com/Tantalor93/dnspyre/master/data/1000-domains
```

```
There was an error while starting benchmark: failed to download file 'https://raw.githubusercontent.com/Tantalor93/dnspyre/master/data/1000-domains' with error 'Get "https://raw.githubusercontent.com/Tantalor93/dnspyre/master/data/1000-domains": tls: failed to verify certificate: x509: certificate signed by unknown authority'
```

**Comment by Tantalor93** [comment]
oh, I missed that scratch container does not have certificates installed, I’ll fix this asap

**Comment by Tantalor93** [comment]
should be fixed in version `v2.12.2`, 
```
docker run  tantalor93/dnspyre:v2.12.2 --duration 10s --server 8.8.8.8 https://raw.githubusercontent.com/Tantalor93/dnspyre/master/data/1000-domains
```
